content/applications/hr/frontdesk/visitors.rst

```diff
@@ -0,0 +1,167 @@
+purposes. This ensures an accurate list of people on the premises.
+
+.. _frontdesk/list:
+
+Visitor list
+============
+
+To access a complete list of checked-in visitors, navigate to :menuselection:`Frontdesk app -->
+Visitors`.
+
+.. note::
+   By default, the :guilabel:`Planned or Checked-In` and :guilabel:`Today` filters appear in the
+   :guilabel:`Search...` bar.
+
+All visitors are presented in a list view, with the following details that were entered upon check
+in:
+
+- :guilabel:`Name`: the guest's name.
+- :guilabel:`Visitor Company`: the company the guest represents.
+- :guilabel:`Phone`: the guest's phone number.
+- :guilabel:`Drinks*`: the drink the guest requested.
+- :guilabel:`Host`: who the guest is waiting to see.
+- :guilabel:`CheckIn`: the date and time the guest checked in.
+- :guilabel:`Checkout*`: the date and time the guest checked out. In the default view, only guests
+  with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible. Guests with check-out
+  times are only visible if the :guilabel:`Today` filter is not active.
+- :guilabel:`Duration`: the amount of time the guest has been checked in for.
+- :guilabel:`Station`: the location of where the guest checked in.
+- :guilabel:`Status`: the status of the guest. The options are :guilabel:`Checked-In`,
+  :guilabel:`Planned`, :guilabel:`Checked-Out`, or :guilabel:`Canceled`.
+- :guilabel:`Email*`: the guest's email address.
+
+\*\ These fields are not visible in the default :guilabel:`Visitor` list. These must be enabled
+using the :icon:`oi-settings-adjust` :guilabel:`(additional options)` icon in the top-right of the
+list.
+
+To the far-right of the titled columns on the :guilabel:`Visitors` page, there is an untitled
+column, where a guest's status can be updated.
+
+When a guest leaves, click the available :guilabel:`Check out` button to update the guest's record,
+and log the date and time they left.
+
+If a scheduled guest arrives, and they did not check in using the *Frontdesk* kiosk, they can be
+checked in here, by clicking the available :guilabel:`Check in` button to log the date and time they
+arrived.
+
+Beside the untitled status column, a :guilabel:`Drink Served` button appears, but only if that
+particular visitor requested a drink.
+
+When their drink has been served, click the :guilabel:`Drink Served` button to indicate the drink
+has been delivered to the guest. Once clicked, that button disappears.
+
+.. image:: visitors/visitors.png
+   :align: center
+   :alt: The full list of currently checked in visitors, with the drinks to be served highlighted.
+
+If any column is not visible, or if a visible column is preferred to be hidden, click the
+:icon:`oi-settings-adjust` :guilabel:`(additional options)` icon, located at the end of the column
+name list.
+
+Clicking that reveals a drop-down menu of column options to enable or disable. A :icon:`fa-check`
+:guilabel:`(check)` icon indicates the column is visible.
```

Comment:
Update icon label, and refrain from "That" language, especially when starting a new paragraph. I was confused what you meant by "Clicking that". Suggesting to restructure to:
```suggestion
:icon:`oi-settings-adjust` :guilabel:`(settings adjust)` icon, located at the end of the column
name list. Doing so, reveals a drop-down menu of column options to enable or disable. A
:icon:`fa-check` :guilabel:`(check)` icon indicates the column is visible.
```